src/test/java/com/google/api/generator/gapic/dummy/FileDiffInfraDummyTest.java

```diff
@@ -36,6 +37,9 @@
   // TODO(xiaozhenliu): remove this test class once the file-diff infra is in place and well-tested.
   private static final String GOLDENFILES_DIRECTORY =
       "src/test/java/com/google/api/generator/gapic/dummy/goldens/";
+  private static final String GOLDENFILES_SIMPLE_CLASS = "FileDiffInfraDummyTestSimpleClass.golden";
```

Comment:
Currently the `GOLDENFILES_SIMPLE_CLASS ` is static variable and `this.getClass().getSimpleName()` for extracting class name needs non-static context. Removed `static` for the variable since it's only used in this class. Thanks